docs/concepts/debugging.md

```diff
@@ -0,0 +1,229 @@
+Debugging Juttle
+================
+
+As with any programming language, coding in Juttle will at some point call for debugging tools to help make sense of a program's behavior.
+
+[TOC]
+
+## Turning On Debug Logging
+
+To run a Juttle program with debug logging on from the CLI, do:
+
+```
+DEBUG=* juttle <program-file.juttle>
+```
+
+or
+
+```
+DEBUG=* juttle -e "juttle-program"
+```
+
+To run programs in the browser environment with debug logging, set `--log-config <log4js-config-path>` and specify desired log level in the log4js config, as described in the [outrigger README](https://github.com/juttle/outrigger).
```

Comment:
Ok, will this be the README with log config? It's currently empty. https://github.com/juttle/juttle-engine